packages/dds/matrix/src/matrix.ts

```diff
@@ -543,33 +644,82 @@ export class SharedMatrix<T = any>
 				);
 				break;
 			default: {
+				if (content.type === MatrixOp.changeSetCellPolicy) {
+					this.submitLocalMessage({
+						type: MatrixOp.changeSetCellPolicy,
+					});
+					break;
+				}
 				assert(
 					content.type === MatrixOp.set,
 					0x020 /* "Unknown SharedMatrix 'op' type." */,
 				);
 
 				const setOp = content as ISetOp<T>;
-				const { rowHandle, colHandle, localSeq, rowsRefSeq, colsRefSeq } =
-					localOpMetadata as ISetOpMetadata;
-
-				// If there are more pending local writes to the same row/col handle, it is important
-				// to skip resubmitting this op since it is possible the row/col handle has been recycled
-				// and now refers to a different position than when this op was originally submitted.
-				if (this.isLatestPendingWrite(rowHandle, colHandle, localSeq)) {
+				const {
+					rowHandle,
+					colHandle,
+					localSeq,
+					rowsRefSeq,
+					colsRefSeq,
+					referenceSeqNumber,
+				} = localOpMetadata as ISetOpMetadata;
+
+				// Policy is not FWW. So behave accordingly.
+				if (this.setCellPolicySwitchOpSeqNumber > -1) {
```

Comment:
I'd recommend to use this.isSetCellConflictResolutionPolicyFWW() instead. Also comment seems wrong - it's FWW ("not" seems like should be removed)